src/schema/v2/sale/__tests__/index.test.ts

```diff
@@ -580,110 +580,129 @@ describe("Sale type", () => {
       [
         {
           auction_state: "open",
-          live_start_at: moment().subtract(1, "days"),
-          registration_ends_at: moment().subtract(2, "days"),
+          live_start_at: moment().subtract(1, "days").toISOString(),
+          registration_ends_at: moment().subtract(2, "days").toISOString(),
         },
         "in progress",
       ],
-      [{ end_at: moment().subtract(1, "days") }, null],
+      [
+        {
+          end_at: moment().subtract(1, "days").toISOString(),
+          start_at: moment().subtract(10, "days").toISOString(),
+        },
+        null,
```

Comment:
Even though the bug was present, this test was incorrectly passing (expected string is L593 here) because `start_at` was not present- the bug does not manifest if the `start_at` value is not present - see [below](https://github.com/artsy/metaphysics/pull/2689/files#diff-6a8cbee18d0733cd05693bceea3c8cd5R83)